pallets/inactivity-tracking/src/lib.rs

```diff
@@ -307,6 +305,7 @@ pub mod pallet {
         /// Allows a collator to mark itself online.
         #[pallet::call_index(3)]
         #[pallet::weight(T::WeightInfo::set_online())]
+        #[allow(clippy::useless_conversion)]
```

Comment:
this looks bad haha

Reply:
That's because the pallet macro doesn't like `DispatchErrorWithPostInfo`, we already have this allow in other pallets.